tools/run-wpt.mjs

```diff
@@ -101,8 +111,6 @@ if (HEADLESS === 'true') {
 const wptRunArgs = [
   '--binary',
   BROWSER_BIN,
-  '--webdriver-binary',
```

Comment:
This arg forced WPT to always run WPT via Mapper

Reply:
Good catch!